moveit_core/trajectory_processing/src/time_optimal_trajectory_generation.cpp

```diff
@@ -891,12 +891,12 @@ bool TimeOptimalTrajectoryGeneration::computeTimeStamps(robot_trajectory::RobotT
     RCLCPP_ERROR(LOGGER, "Failed to get active variable indices.");
   }
 
-  const size_t num_joints = indices.size();
-  Eigen::VectorXd max_velocity(num_joints);
-  Eigen::VectorXd max_acceleration(num_joints);
-  for (const auto idx : indices)
+  const size_t num_active_joints = indices.size();
+  Eigen::VectorXd max_velocity(num_active_joints);
+  Eigen::VectorXd max_acceleration(num_active_joints);
+  for (size_t idx = 0; idx < num_active_joints; ++idx)
```

Comment:
Just like Peter, I was confused initially. Let's improve the naming:

```suggestion
  for (size_t active_joint_idx = 0; active_joint_idx < num_active_joints; ++active_joint_idx)
```